hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+    Basically it needs higher order(5-order) low pass filter since the fundamental power is very high(+30dBm).
+
+  - Rx requirements
+    
+    No sensitivity requirement. But the limits for harmonics or spurs in a restricted band is defined in “FCC Part 15.209 Unintentional Radiators” as following:
+    - Spurious falls below 960 MHz = -49.2 dBm EIRP (max) (15.209.a);
+    - Spurious falls above 960 MHz = -41.2 dBm EIRP (max) (15.209.a).
+
+- Part-15.249
+  It is for proprietary application. It covers 2.4GHz band.
+  - Tx power limits
+   50 mV/meter @ 3 meters → -1.2 dBm EIRP (max) (15.249.a); It is measured by using an averaging detector above 1 GHz (15.35.b)
+
+  - Tx bandwidth requirement
+   For such low power, FCC doesn’t care if energy isn’t “spread out”. so there is no band limit.
+   
+  - Spurs emission limits
+    For all harmonics, allowed 500 uV/meter @ 3 meters → -41.2 dBm EIRP (max) (15.249.a); the other spurs should not be greater than -50dBc of fundamental or -41.2dBm, which ever is the lesser attenuation. All the spurs strength limits can be relax by using DCF factor.Because radiated spurious emissions above 1 GHz are AVERAGE measurements covered by FCC Part 15.35.
+    Basically it doesn't need higher order(5-order) low pass filter for attenuating harmonicssince since the fundamental power is low. A 3-order low pass filter is enough.
+
+  - Rx requirements
+    
+    No sensitivity requirement. But the limits for harmonics or spurs in a restricted band is defined in “FCC Part 15.109 Unintentional Radiators” as following:
+    - Spurious falls below 960 MHz = -49.2 dBm EIRP (max) (15.209.a);
+    - Spurious falls above 960 MHz = -41.2 dBm EIRP (max) (15.209.a).
+
+## 4.3. CE requirements in Sub-GHz bands
+  ETSI(European Telecommunications Standards Institute) [EN300220-01](https://www.etsi.org/deliver/etsi_en/300200_300299/30022001/03.01.01_60/en_30022001v030101p.pdf) mainly defines the sub-GHz regulation for CE certification. And it regulates the 25 MHz to 1000 MHz Frequency Range with Power Levels up to 500mW. Silicon Labs' product mainly focus in 169MHz, 434MHz, and 868MHz operating bands. The typical applications are Control links, Alarms, and Metering in these sub-GHz bands. 
+ - Tx power limits
+   Tx power can be conducted measurement for devices with antenna connectors, and radiated measurement for devices with integral antennas. The power limits are listed in radiation ERP. 
+    - +27dBm at 169.4~169.475MHz(Meter Reading)
+    - +10dBm at 434.04~434.79MHz
+    - +27dBm at 869.4~869.65MHz(Narrow band)
+
+ - Frequency error
+  Frequency error is the frequency difference between the measured unmodulated carrier frequency and the manufacturer stated normal frequency. The frequency error shall not exceed the following table under normal and extreme conditions:
+  <div align="center">
+  <img src="files\HW-Comply-to-the-Regulations\Frequency-Error.png">  
```

Comment:
Where was this image referenced from?  We should provide proper reference to the document where this was referenced.

Reply:
Yes, it will add the reference.

Reply:
Added the reference.